src/three-components/ModelScene.ts

```diff
@@ -49,27 +61,47 @@ const $modelAlignmentMask = Symbol('modelAlignmentMask');
  * Provides lights and cameras to be used in a renderer.
  */
 export default class ModelScene extends Scene {
+  private[$paused]: boolean = false;
+  private[$modelAlignmentMask] = new Vector3(1, 1, 1);
+  private aspect: number;
+
+  public canvas: HTMLCanvasElement;
+  public light: HemisphereLight;
+  public renderer: Renderer;
+  public shadowLight: DirectionalLight;
+  public shadow: StaticShadow;
+  public pivot: Object3D;
+  public width: number;
+  public height: number;
+  public framedHeight: number = 1;
+  public modelDepth: number = 1;
+  public isVisible: boolean = false;
+  public isDirty: boolean = false;
+  public element: ModelViewerElementBase;
+  public context: CanvasRenderingContext2D;
+  public exposure: number;
+  public model: Model;
+  public camera: PerspectiveCamera;
+  public activeCamera: Camera;
+
+
   /**
    * @param {ModelViewerElementBase} options.element
    * @param {CanvasHTMLElement} options.canvas
    * @param {number} options.width
    * @param {number} options.height
-   * @param {THREE.WebGLRenderer} options.renderer
+   * @param {Renderer} options.renderer
```

Comment:
Are these type comments still necessary now that they're defined above?

Reply:
They can be removed. I filed https://github.com/GoogleWebComponents/model-viewer/issues/581